pxr/usd/usdShade/validators.cpp

```diff
@@ -590,6 +589,244 @@ _SubsetsMaterialBindFamily(const UsdPrim& usdPrim)
     return errors;
 }
 
+static
+UsdValidationErrorVector
+_NormalMapTextureValidator(const UsdPrim& usdPrim) {
+
+    if (!usdPrim.IsA<UsdShadeShader>()) {
+        return {};
+    }
+
+    const UsdShadeShader shader(usdPrim);
+    if (!shader) {
```

Comment:
@tallytalwar I see this check here: https://github.com/PixarAnimationStudios/OpenUSD/blob/dev/pxr/usd/usdUtils/complianceChecker.py#L468 however I'm not sure what USD structure would trigger this failure, after it's already passed the `IsA` check.

Reply:
Yeah I don't think so also.